### DISCUSSION TITLE
animal_mt: Warning: No paired reads found!? ends with incomplete assembly

### DISCUSSION THREAD

**Discussion by ResidentialDragonSlayer** [Kinggerm/GetOrganelle#290] [Q&A] [answered]
I am trying to assemble a mitochondrial genome using data from GenBank's SRA. Initially the file was interleaved, but when I failed to figure out how to run GetOrganelle on interleaved data I separated it into paired files. I have tried twice with slightly different parameters (second time I set `--reduce-reads-for-coverage inf` and `-s` for a similar species). Both times, after extending, the program has warned that no paired reads were found, and disentangling has failed. Can you tell me what might be wrong?
Log from the first run: 
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/12303944/get_org.log.txt)
Log from the second run:
[get_org.log.txt](https://github.com/Kinggerm/GetOrganelle/files/12303947/get_org.log.txt)

Format of initial fastq files:
@SRR2103277.1/1 SN860:429:H9CNTADXX:1:1101:10712:1988 length=143
AAGTCCCTAATCGCGTACTCCTCCGTAAGTCACATAGCCCTAGTGATCGCAGCGGTACTAATCCAGACCCNNNNANNNTACGCAGGAGCAACAGCCCTAATGATTGCCCANGGTNTAANNNNNNCNANGCTANNCNNCCTAGC
+
???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
@SRR2103277.2/1 SN860:429:H9CNTADXX:1:1101:1372:2224 length=143
ACTCACAAAATCTCCCATTAAAGAAAATCATATTGTAATTAATAGCTGTTGAAGCTCCTGGAGCCCCTGCGTGGCTCAGTGGGTTGAAGCCTCTGCCTTCGGCTCGGGTCATGATCTCAGGGTCCTGGGATCAAGCCCCACAT
+
???????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????????

Thank you!

**Comment by JianjunJin** [comment, marked as answer]
The `No paired reads found` issue is probably because that the data does not contain pairing information or GetOrganelle did not recognize it well. Anyway, it is not a big issue here and is not the reason for disentangling failure. 

The disentangling failure is a classic problem and you may first check https://github.com/Kinggerm/GetOrganelle/wiki/FAQ#what-should-i-do-with-incomplete-resultbroken-assembly-graph

**Reply by ResidentialDragonSlayer** [reply]
Thank you for your response. Would improperly processing the reads as unpaired affect the quality of the final assembly?
I will look into what to do with an incomplete result.

**Reply by ResidentialDragonSlayer** [reply]
Thank you, I adjusted the word size and it worked. My homemade paired files were indeed formatted wrong, thank you for pointing that out as well!
